### DISCUSSION TITLE
✨ Inspirations ✨

### DISCUSSION THREAD

**Discussion by anomatomato** [AuDojo/AuDojo#144] [Show and tell]
Get inspired by other websites!

**Comment by trang-ngn** [comment]
https://csvistool.com

algorithm and datastructure visualizer  (cool icons)

![image](https://github.com/user-attachments/assets/5e26c23a-2e4a-4bdc-866f-2e2b6d109f30)

**Comment by trang-ngn** [comment]
https://github.com/adimail/visual-data-structures?tab=readme-ov-file

(cool buttons)

![image](https://github.com/user-attachments/assets/9240194e-5ebb-4b3a-9666-aff74299cd2a)

**Comment by anomatomato** [comment]
https://www.sortvisualizer.com/

![image](https://github.com/user-attachments/assets/1dc35e7d-c4c5-4fc9-b49e-de180d69b3b8)

**Comment by trang-ngn** [comment]
https://webad.univie.ac.at/webad/htmls/trees/avl.html (with canvas)

<img width="1280" alt="Bildschirmfoto 2025-01-02 um 17 33 05" src="https://github.com/user-attachments/assets/6dab03fc-3b7c-44ca-b6a7-42686de46dfe" />

**Comment by trang-ngn** [comment]
https://saketthota.github.io/AVL-Tree-Visualizer/ (with canvas)
<img width="1280" alt="Bildschirmfoto 2025-01-02 um 17 40 09" src="https://github.com/user-attachments/assets/f515a56d-6eed-42ca-9244-0dfaa2121174" />

**Comment by anomatomato** [comment]
AVL Tree Vis with d3
(Source code included!)

https://github.com/prachivishnoi27/AVL-Visualization

![image](https://github.com/user-attachments/assets/b46f76af-c353-45b4-89f7-f6cacef7532b)
